setup.cfg

```diff
@@ -24,6 +24,7 @@ classifiers =
 packages = find:
 zip_safe = False
 tests_require = pytest-astropy
+# setup_requires is deprecated in favor of pyproject.toml and will be removed
 setup_requires = setuptools_scm
```

Comment:
I say burn it, I don't think this is having any useful effect.

```suggestion
```

Reply:
Done! You're right, no discernible effect.